### PR TITLE
Deprecate sourced transfers

### PR DESCRIPTION
**Pull request by jnovak-stripe**
Removed sourced_transfers from balance transaction resource because that field has been deprecated in new versions of the API.

Having some Travis trouble so trying a new PR. Original PR is https://github.com/stripe/stripe-java/pull/343

### CODE REVIEW

**Comment by jnovak-stripe** [comment]
I can't explain why this passes but the other PR didn't. My hypothesis is that I did a git rebase that confused Travis. Closing the other PR.

r? @brandur

File: src/main/java/com/stripe/model/BalanceTransaction.java [hunk 1]
```diff
@@ -116,6 +118,7 @@ public void setSource(String source) {
 		this.source = source;
 	}
```
Comment (ob-stripe) [inline src/main/java/com/stripe/model/BalanceTransaction.java]: Could you add a Javadoc comment here? Something like this:

```java
/**
 * @deprecated
 * Recent API versions no longer return this field (https://stripe.com/docs/upgrades#2017-01-27).
 * Prefer listing all transfers with the `transfer_group` parameter: https://stripe.com/docs/api/java#list_transfers-transfer_group.
 */
```

Feel free to reword this as you think is appropriate.

Comment (jnovak-stripe) [inline src/main/java/com/stripe/model/BalanceTransaction.java]: added

**Comment by brandur-stripe** [comment]
Makes sense. Thank you!

LGTM.

**Comment by brandur-stripe** [comment]
Released as 3.10.0.
